### PR TITLE
Updated Harmony Guide to make it more clearer for new comers

### PR DESCRIPTION
**Pull request by johnaqu1no**

### CODE REVIEW

File: docs/basic/harmony_guide.md [hunk 1]
```diff
@@ -23,21 +23,50 @@ of the class you are patching).
...
     }
 }
 ```
+
+## Debugging
+Before testing your mod, I highly recommend for you to turn on bepinex's console for Among Us.  
+Head on to the path: `C:\Program Files (x86)\Steam\steamapps\common\Among Us\BepInEx\config`  
+Click `BepInEx.cfg` and find the header `[Logging.Console]` and change `Enabled = false` to `Enabled = true`
+
+## Running 4 instances with just 2 clicks
```
Comment (js6pak) [inline docs/basic/harmony_guide.md]: Windows only and its looking sus in general

File: docs/basic/harmony_guide.md [hunk 2]
```diff
@@ -23,21 +23,50 @@ of the class you are patching).
...
+If you want to run 4 instances of Among Us without clicking the exe file 8 times, you can download this .bat file and place it inside:  
+`C:\Program Files (x86)\Steam\steamapps\common\Among Us`  
+Download the .bat file: https://cdn.discordapp.com/attachments/790517195003527189/816540451062153226/AUTester.bat
+
+## Reactor.Debugger
+If you wish to start the game with 3 clients, you can use the `Reactor.Debugger.dll`  
+Click this link: https://github.com/NuclearPowered/Reactor/suites/2107184644/artifacts/42856716
```
Comment (js6pak) [inline docs/basic/harmony_guide.md]: This is a direct link that will get outdated, nightly.link is a cool alternative https://nightly.link/NuclearPowered/Reactor/workflows/main/master/Reactor.Debugger.dll

**Review by js6pak — CHANGES_REQUESTED** [review]

File: docs/basic/harmony_guide.md (hunk 1 above)
Comment (johnaqu1no) [reply docs/basic/harmony_guide.md]: you're sus

**Comment by johnaqu1no** [comment]
Updated @js6pak

**Comment by js6pak** [comment]
Debugging isn't windows only, "Running 4 instances with just 2 clicks" bat thing is

**Comment by johnaqu1no** [comment]
> Debugging isn't windows only, "Running 4 instances with just 2 clicks" bat thing is

Well, I assumed you meant the path was for windows only.

**Comment by johnaqu1no** [comment]
anything else needed?

**Pull request merged**